apps/dashboard/app/javascript/turbo_shim.js

```diff
@@ -19,3 +19,15 @@ export function replaceHTML(id, html) {
     ele.innerHTML = newHTML;
   }
 }
+
+export function pollAndReplace(url, delay, id) {
+  fetch(url, { headers: { Accept: "text/vnd.turbo-stream.html" } })
+    .then(response => response.ok ? Promise.resolve(response) : Promise.reject(response.text()))
+    .then((r) => r.text())
+    .then((html) => replaceHTML(id, html))
+    .then(setTimeout(pollAndReplace, delay, url, delay, id))
+    .catch((err) => {
+      console.log('Cannot not retrive partial due to error:');
```

Comment:
This all looks good, but it's early, and I swear this is reading as a double negative (but I'm brewing my coffee still). Should this just be `Cannot retrieve partial due to error:` or am I still waking up?

Reply:
Lol yea, that's a double negative. I'll update now.